Makefile.source.mjs

```diff
@@ -264,6 +264,7 @@ target["prepublish-build"] = function () {
       target["prepublish-prepare-dts"]();
     },
     {
+      NODE_ENV: "production",
```

Comment:
Can we unify `NODE_ENV` into `BABEL_ENV`? There is currently only one place where `NODE_ENV` is used, and there are more `BABEL_ENV`.

Reply:
Could you open a PR to replace both? :pray:

Reply:
Yeah. I'll do this after `7.19.0` is released, it's a little messy and I'm worried about causing accidents.